packages/moonstone/IncrementSlider/IncrementSlider.js

```diff
@@ -516,12 +497,11 @@ const IncrementSliderBase = kind({
 		return (
 			<div {...rest}>
 				<IncrementSliderButton
+					{...voiceProps}
```

Comment:
Are there any `data-webos-voice-` props that can't be applied to multiple components with the same value?

Reply:
Thanks for mention about this. As you said, some voice props should not be delegated to IconButtons. So, I did update commit for fix it.